File: project-euler-100-java/src/test/java/com/github/adrianomoreira/projecteuler100/Problem002Test.java
package com.github.adrianomoreira.projecteuler100;

import com.github.adrianomoreira.projecteuler100.Problem002;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Problem002Test {

    @DisplayName("fiboEvenSum(10) should return 44")
    @Test
    public void test() {
        assertEquals(44, Problem002.fiboEvenSum(10));
    }

    @DisplayName("fiboEvenSum(18) should return 3382")
    @Test
    public void test2() {
        assertEquals(3382, Problem002.fiboEvenSum(18));
    }

    @DisplayName("fiboEvenSum(23) should return 60696")
    @Test
    public void test3() {
        assertEquals(60696, Problem002.fiboEvenSum(23));
    }

    @DisplayName("fiboEvenSum(43) should return 350704366")
    @Test
    public void test4() {
        assertEquals(350704366, Problem002.fiboEvenSum(43));
    }
//    Your function should return an even value.%

}

File: project_euler_100_dart/test/problem_002_test.dart
import 'package:flutter_test/flutter_test.dart';

import 'package:project_euler_100_dart/problem_002.dart';

void main() {
  test("fiboEvenSum(10) should return 44", () {
    expect(fiboEvenSum(10), 44);
  });
  test("fiboEvenSum(18) should return 3382", () {
    expect(fiboEvenSum(18), 3382);
  });
  test("fiboEvenSum(23) should return 60696", () {
    expect(fiboEvenSum(23), 60696);
  });
  test("fiboEvenSum(43) should return 350704366", () {
    expect(fiboEvenSum(43), 350704366);
  });
}
